source/options.css

```diff
@@ -1,6 +1,11 @@
+@media not screen and (width: 400px) { /* Excludes Edge */
```

Comment:
Was this query tested in Edge?

Reply:
>It works in both Edge and Chrome 😄

_Originally posted by @kidonng in https://github.com/sindresorhus/refined-github/issues/3312#issuecomment-653560583_

Reply:
Tested